packages/@aws-cdk/pipelines/README.md

```diff
@@ -144,6 +144,21 @@ new MyPipelineStack(this, 'PipelineStack', {
 });
 ```
 
+Deployment is done by default with `CodePipeline` engine by first generating
```

Comment:
Is this the most logical place in this README to add this content? I feel a separate section might be better.